tests/backends/test_utils.py

```diff
@@ -90,3 +91,25 @@ def test_unsupported_callproc_kparams_raises_error(self):
         with self.assertRaisesMessage(NotSupportedError, msg):
             with connection.cursor() as cursor:
                 cursor.callproc('test_procedure', [], {'P_I': 1})
+
+
+class CursorDebugWrapperTests(SimpleTestCase):
+    @mock.patch("django.db.backends.utils.logger")
```

Comment:
Please use single quotes throughout.

Reply:
I'm surprised there isn't a linting test for this kind of thing. Seems a waste of brain cycles for code reviewers to have to look for stylistic errors.